dbschema.sql

```diff
@@ -8,55 +7,70 @@ CREATE TABLE LICCB.users(
     userEnabled BOOLEAN NOT NULL
 );
 
-CREATE TABLE LICCB.history(
-    participantID   CHAR(36) NOT NULL PRIMARY KEY,
-    firstName       VARCHAR(30) NOT NULL,
-    lastName        VARCHAR(30) NOT NULL,
-    eventCount      INT DEFAULT 0 NOT NULL,
-    cancellations   INT DEFAULT 0 NOT NULL,
-    lateCancel      INT DEFAULT 0 NOT NULL,
-    volunteerCount  INT DEFAULT 0 NOT NULL,
-    thumbsCount     INT DEFAULT 0 NOT NULL,
-    notes           TEXT
-);
-
 CREATE TABLE LICCB.events(
+    -- IDs
     eventID         CHAR(36) NOT NULL PRIMARY KEY,
+    managerID       CHAR(36) NOT NULL,              -- 'userID' of the user assigned to the event
+    creatorID       CHAR(36) NOT NULL,              -- 'userID' of the user who created the event
+
+    -- Event Details
     eventName       VARCHAR(100) NOT NULL,
-    manager         CHAR(36) NOT NULL,
-    creatorID       CHAR(36) NOT NULL,
-    capacity        INT NOT NULL,
     maxPartySize    INT NOT NULL,
     privateEvent    BOOLEAN NOT NULL,
     startTime       DATETIME NOT NULL,
     endTime         DATETIME NOT NULL,
-    staffRatio      FLOAT NOT NULL,
-    published       BOOLEAN NOT NULL,
-    eventNotes      TEXT,
```

Comment:
Why remove event notes?

Reply:
Thought it would be replaced by eventDesc but it really isn't so I'll add that back.